website/content/en/docs/deployment/vanilla/guide.md

```diff
@@ -1,28 +1,20 @@
 +++
-title = "Vanilla Installation"
-description = "Deploy Kubeflow on AWS using Amazon Elastic Kubernetes Service (EKS)"
-weight = 30
+title = "Manifest Deployment Guide"
+description = "Deploy the vanilla distribution of Kubeflow on AWS using Kustomize or Helm"
+weight = 20
 +++
 
-# Deploying Kubeflow on EKS
-
-This guide describes how to deploy Kubeflow on AWS EKS. This vanilla version has minimal changes to the upstream Kubeflow manifests.
+This guide describes how to deploy Kubeflow on Amazon EKS. This vanilla version has minimal changes to the upstream Kubeflow manifests.
 
 ## Prerequisites
 
-Be sure that you have satisfied the [installation prerequisites]({{< ref "../prerequisites.md" >}}) before working through this guide.
-
-### Build Manifests and install Kubeflow
-
-There two options for installing Kubeflow official components and common services with kustomize.
-
-1. Single-command installation of all components under `apps` and `common`
-2. Multi-command, individual components installation for `apps` and `common`
+Be sure that you have satisfied the installation prerequisites before working through this guide. You need to:
+- [Set up your deployment environment]({{< ref "prerequisites.md" >}})
+- [Create an EKS Cluster]({{< ref "create-eks-cluster.md" >}})
 
-Option 1 targets ease of deployment for end users. \
-Option 2 targets customization and ability to pick and choose individual components.
+## Build Manifests and install Kubeflow
 
-> Warning: In both options, we use a default email (`user@example.com`) and password (`12341234`). For any production Kubeflow deployment, you should change the default password by following [the relevant section](#change-default-user-password).
+> ⚠️ Warning: We use a default email (`user@example.com`) and password (`12341234`) for our guides. For any production Kubeflow deployment, you should change the default password by following the steps in [Change default user password]({{< ref "../connect-kubeflow-dashboard#change-the-default-user-password-kustomize" >}}).
```

Comment:
remove the Note below this, we are changing the behaviour in this release

@ryansteakley